backend/src/data_entry/mod.rs

```diff
@@ -252,7 +256,20 @@ pub async fn polling_station_data_entry_finalise(
         ));
     }
 
-    polling_station_data_entries.finalise(&mut tx, id).await?;
+    match entry_number {
+        1 => {
+            polling_station_data_entries
+                .finalise_first_entry(&mut tx, id)
+                .await?
+        }
+        2 => polling_station_data_entries.finalise(&mut tx, id).await?,
```

Comment:
This needs to be the same for both the first and second data entry.

The creation of the data entry result needs to be triggered after the second data entry is finalised. In a next issue, we need to add a condition to only do this when the first and second data entry are equal. For this issue, we can skip that check.

Reply:
Fixed in [ab34fce](https://github.com/kiesraad/abacus/pull/612/commits/ab34fced8a4d57f0effbf4ffc9f5e0b8a861c392).

I verified that the frontend still works as-is by clicking through an election.